app/views/interview_questions/index.html.slim

```diff
@@ -0,0 +1,24 @@
+- provide(:title, 'All interview questions')
+.container.content-area
+  h1
+    | All interview questions
+  .row  
+    = render 'layouts/messages'
+  br
+  table.table.table-bordered.table-hover
+    tr.active
+      th width="5%"
```

Comment:
tag attributes can not be used for changing layout.
Use css instead. Find and fix such mistakes in this PR.